src/internal/operations/backup.go

```diff
@@ -294,8 +294,10 @@ func (op *BackupOperation) do(
 // checker to see if conditions are correct for incremental backup behavior such as
 // retrieving metadata like delta tokens and previous paths.
 func useIncrementalBackup(sel selectors.Selector, opts control.Options) bool {
-	// TODO(meain): remove this once we stabilize delta incrementals for OneDrive
-	if sel.Service == selectors.ServiceOneDrive {
+	// TODO(meain): remove this once we stabilize delta incrementals for OneDrive.
+	// Updated version will still need to check if permissions backup is enabled
+	// though because we can't support that and delta incrementals right now.
+	if sel.Service == selectors.ServiceOneDrive && !opts.ToggleFeatures.EnablePermissionsBackup {
```

Comment:
Does this need any customer facing documentation?  Or, at least, a "known issues" entry?

Reply:
probably yes, though I'm also not exactly sure how this is planned to be released right now. It's based on a PR that adds a separate flag for OneDrive, but I'm not sure if we'll release the extra flag or not

Reply:
We should add an entry to "known issues" section about this in changelog though.